src/tools/schema2graph.ts

```diff
@@ -132,12 +127,12 @@ export class SchemaGraph {
 
     // If this node only has one alias, use that for the class name.
     // Otherwise generate an internal name and create aliases for it.
-    if (node.aliases.length === 1) {
-      node.name = node.aliases.pop();
-      node.kotlinName = node.name;
+    if (node.connections.length === 1) {
+      node.name = this.nameGenerator(node);
+      node.aliases = this.aliasGenerator(node);
```

Comment:
This doesn't seem right - there shouldn't be any aliases when there's only one instance of a schema.

Reply:
In Kotlin we are using the aliases as a transition mechanism so we don't have to update all the Kotlin particles and cause copybara madness every time we improve the naming. This way we can update the entity class names and rely on the aliases. Then we can update the particles before removing the Kotlin aliases.

Reply:
Ok, so this is an interim step?

Reply:
Yes, correct.

Reply:
:+1: